jekyll/_cci2/server-3-operator-vm-service.adoc

```diff
@@ -14,7 +14,7 @@ version:
 CircleCI server's VM service controls how https://circleci.com/docs/2.0/executor-types/#using-machin[`machine`] executor
 (Linux and Windows images) and https://circleci.com/docs/2.0/building-docker-images[Remote Docker] jobs are run.
 
-This section describes the available configuration options for VM service. These config options are all accessible from the KOTS admin console by choosing the Config tab frm your dashboard.
+This section describes the available configuration options for VM service. These config options are all accessible from the KOTS admin console by choosing the Config tab from your dashboard.
```

Comment:
```suggestion
This section describes the available configuration options for the VM service. These config options are all accessible from the KOTS admin console by choosing the Config tab from your dashboard.
```

Reply:
Not sure if this is supposed to be "the VM service" or simply "VM service."

Reply:
VM service is the name of a service within the server installation so I think it's OK as is.